### PR TITLE
Adding a template for deploying headless apps.

### PR DESCRIPTION
**Pull request by LindsayYoung**
These are used for long running tasks.

### CODE REVIEW

File: manifests/manifest_task_dev.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    NEW_RELIC_CONFIG_FILE: newrelic.ini
+    NEW_RELIC_ENV: development
+    NEW_RELIC_LOG: stdout
+    PRODUCTION: True
+    WEB_CONCURRENCY: 4
+  no-route: true
+  services:
+  - fec-redis-dev
+  - fec-s3-dev
+  - fec-eregs-search
```
Comment (vrajmohan) [inline manifests/manifest_task_dev.yaml]: This should be fec-api-search

Comment (LindsayYoung) [inline manifests/manifest_task_dev.yaml]: Thanks!

**Comment by vrajmohan** [comment]
Sorry for the late suggestion - should we rename the files to manifests/manifest_task_{space}.yaml.template and make them read-only. This will also prevent accidental checkins.

**Comment by LindsayYoung** [comment]
No worries, that is a good idea!

**Comment by vrajmohan** [comment]
Nice work!

**Pull request merged**